.eslintrc.js

```diff
@@ -77,6 +78,38 @@ module.exports = {
     'unicorn/no-array-push-push': 'error',
     'unicorn/prefer-keyboard-event-key': 'error',
     'unicorn/prefer-regexp-test': 'error',
-    'unicorn/prefer-string-replace-all': 'error'
+    'unicorn/prefer-string-replace-all': 'error',
+    // '@intlify/vue-i18n/no-dynamic-keys': 'error',
+    // '@intlify/vue-i18n/no-unused-keys': [
+    //   'error',
+    //   {
+    //     extensions: ['.js', '.vue']
+    //   }
+    // ],
```

Comment:
Might want to add some comment briefly explaining why they are added in commented form